source-repo-scripts/release_pull_request.bash

```diff
@@ -14,28 +14,33 @@
 # limitations under the License.
 #
 
-# The script will open a pull request for a forward port.
+# The script will open a pull request a release PR (actually opens the PR page in your web browser)
 #
 # Requires the 'gh' CLI to be installed.
 #
 # Usage:
-# $ ./merge_forward_pull_request.bash <from_branch> <to_branch>
+# $ ./release_pull_request.bash <from_branch> <to_branch>
 #
-# For example, to merge `ign-rendering6` forward to `main`:
+# For example, to release `gz-rendering7` 7.1.0
 #
-# ./merge_forward_pull_request.bash ign-rendering6 main
+# ./release_pull_request.bash 7.1.0 gz-rendering7
+#
+# Make sure you've checked out the branch that has the changes for the release
+# and that the changes have been pushed.
 
 VERSION=${1}
 TO_BRANCH=${2}
-PREV_VER=${3}
 
-if [[ $# -ne 3 ]]; then
-  echo "./release_pull_request.bash <version> <to_branch> <previous_version>"
+if [[ $# -ne 2 ]]; then
+  echo "./release_pull_request.bash <version> <to_branch>"
   exit 1
 fi
 
 set -e
 
+git fetch --tags
+PREV_VER=$(git describe --tags --abbrev=0 | sed 's/.*_//')
```

Comment:
I'm tempted to try to use the `TO_BRANCH` variable here to try to make sure we identify a tag that matches the major version of our target branch, though it may not be necessary. Feel free to ignore this suggestion. 

```suggestion
PREV_VER=$(git describe --tags --match ${TO_BRANCH}* --abbrev=0 | sed 's/.*_//')
```

Reply:
+1